Src/Persistence/Migrations/Logging.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace TeamProject.Persistence.Migrations
+{
+    public partial class Logging : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<string>(
+                "Message",
+                "Logs",
+                maxLength: 512,
+                nullable: false,
+                oldClrType: typeof(string),
+                oldType: "nvarchar(512)",
+                oldMaxLength: 512,
+                oldNullable: true);
+
+            migrationBuilder.AlterColumn<string>(
+                "Level",
+                "Logs",
+                maxLength: 64,
+                nullable: false,
+                oldClrType: typeof(string),
+                oldType: "nvarchar(64)",
+                oldMaxLength: 64,
+                oldNullable: true);
+
+            migrationBuilder.AlterColumn<string>(
+                "Application",
+                "Logs",
+                maxLength: 64,
+                nullable: false,
+                oldClrType: typeof(string),
+                oldType: "nvarchar(64)",
+                oldMaxLength: 64,
+                oldNullable: true);
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<string>(
+                "Message",
+                "Logs",
+                "nvarchar(512)",
+                maxLength: 512,
+                nullable: true,
+                oldClrType: typeof(string),
+                oldMaxLength: 512);
+
+            migrationBuilder.AlterColumn<string>(
+                "Level",
+                "Logs",
+                "nvarchar(64)",
+                maxLength: 64,
+                nullable: true,
+                oldClrType: typeof(string),
+                oldMaxLength: 64);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assign this magic number '64' to a well-named (variable|constant), and use the (variable|constant) instead.](https://app.codacy.com/manual/liannoi/team-project/pullRequest?prid=5600534)